explore/src/main/scala/explore/targeteditor/ElevationPlotSection.scala

```diff
@@ -142,9 +142,9 @@ object ElevationPlotSection:
                 .map(BoundedInterval.fromInterval)
                 .flattenOption
 
-        <.div(ExploreStyles.ElevationPlotSection)(
+        React.Fragment(
           HelpIcon("target/main/elevation-plot.md".refined, ExploreStyles.HelpIconFloating),
-          <.div(ExploreStyles.ElevationPlot) {
+          <.span(ExploreStyles.ElevationPlot)(
```

Comment:
Why a `span`?

Reply:
I was just trying things, this can be reverted, I'll push a commit